### PR TITLE
Character counter for counting characters

### PR DESCRIPTION
**Pull request by mrjoelkemp**
This would make one sick ractive component.

### CODE REVIEW

File: test/specs/ux/CharCounter.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+define([
+  'jquery',
+  'ux/CharCounter',
+  'text!fixtures/ux/CharCounter.html'
+], function($, CharCounter, fixture) {
+  'use strict';
+
+  describe('ux/CharCounter', function() {
+    beforeEach(function() {
+      $('#jasmine_content').append(fixture);
```
Comment (mikesherov) [inline test/specs/ux/CharCounter.js]: no setFixture call?

Comment (mrjoelkemp) [inline test/specs/ux/CharCounter.js]: Facemailed. That's provided by jasmine-jquery which isn't available. Introducing it caused other tests to fail; I avoided that yak.

**Comment by mikesherov** [comment]
:+1:

**Comment by Aintaer** [comment]
:+1:

**Pull request merged**